H_ecto_models.md

```diff
@@ -597,6 +597,73 @@ end
 ```
 Because we declared a relationship in `HelloPhoenix.User`, `%User{}` will now contain a videos property which starts out as an unloaded relationship. In order to properly display it in `render`, we'll need to tell Ecto to preload the field. Note that `Repo.preload/2` is smart enough to work on just one model or collection of them.
 
+### Working with Callback Hooks
+
+Continuing with our video sharing app example, suppose some of our content-uploading users are horrible trolls and one of their favorite pastimes is to initially upload a video of happy huskies playing with furry kittens, then, after gathering a large number of "likes", change the happy video to an extremely controversial and blatantly offensive one. Then, they would write an inflammatory article about how the users of our app "like" extremely offensive videos, and spread this article around on social media to damage our online reputation and maybe even destroy business.
```

Comment:
I appreciate the humor. :) If we want to match the tone of the rest of the guide, though, we should probably re-think these first two paragraphs. :)

Here's an idea. This is just an idea. What if we focused on the `views` column instead? We could ask readers to imagine an `event` model with `type`, `model`, and `model_id` columns. Whenever a view event happens, an after_insert callback fires to update this column.

We wouldn't need to generate a whole new model because we aren't showing how the code behaves in a live system. We could show the event model with the schema and the callback.

What do you think?